packages/aggregator/src/analytics/aggregateAnalytics/aggregations/index.js

```diff
@@ -17,3 +17,4 @@ export { sumAcrossPeriods } from './sumAcrossPeriods';
 export { sumEachDataElement } from './sumEachDataElement';
 export { sumPreviousPerPeriod } from './sumPreviousPerPeriod';
 export { replaceOrgUnitWithOrgGroup } from './replaceOrgUnitWithOrgGroup';
+export { layerYearOnYear } from './layerYearOnYear';
```

Comment:
I'm quite confused... this file doesn't exist? Is it meant to be imported from `utils`?